### PR TITLE
Linted cmd/ packages. Refs #4098

### PR DESCRIPTION
**Pull request by Gouthamve**
- [x] CHANGELOG.md updated
- [x] Rebased/mergable
- [x] Tests pass
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Part of #4098.

### CODE REVIEW

**Comment by Gouthamve** [comment]
While I am confident about all others, I am not really sure if the comments in this commit make sense:
https://github.com/Gouthamve/influxdb/commit/c9d48c0c9da6cf97ee7381f04b6838e740090e53

File: cmd/influx_tsm/converter.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/influxdata/influxdb/tsdb/engine/tsm1"
 )
 
+// KeyIterator is the interaface to iterate over b*1 keys for conversion to tsm1 keys
```
Comment (jsternberg) [inline cmd/influx_tsm/converter.go]: Interface is spelled wrong. I also think you can change the wording here so you aren't saying "this is an interface". You can just say something like `KeyIterator is used to iterate over b* keys for conversion to tsm keys`.

Also avoid referring to `tsm` as `tsm1`. The version number isn't important.

File: cmd/influx_tsm/main.go [hunk 2]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/influxdata/influxdb/cmd/influx_tsm/tsdb"
 )
 
+// ShardReader is the interaface which reads b*1 shards and converts to tsm1 shards
```
Comment (jsternberg) [inline cmd/influx_tsm/main.go]: Here too.

**Comment by jsternberg** [comment]
Other than the two issues I pointed out, this looks fine. Just find out why the tests are failing and squash the commits into one if you can and I'll give a +1.

**Comment by Gouthamve** [comment]
Fixed those issues! And the tests are failing on a different package altogether due network i/o timeouts

`FAIL	github.com/influxdata/influxdb/cluster	1.027s`

**Comment by jsternberg** [comment]
The tests work for me locally so I figure that's good enough.

:+1:

**Pull request merged**